src/agents/agent.py

```diff
@@ -151,7 +151,7 @@ def make_decision(self, prompt: str, project_name: str) -> str:
                     project_name_space_url)
                 response = f"I have generated the PDF document. You can download it from here: {pdf_download_url}"
 
-                #asyncio.run(self.open_page(project_name, pdf_download_url))
+                asyncio.run(self.open_page(project_name, pdf_download_url))
```

Comment:
This will redo the changes in "Fix PDF download" PR and will break the PDF download feature